kolibri/plugins/style_guide/assets/src/views/content/text-fields/example.html

```diff
@@ -0,0 +1,44 @@
+<template>
+
+  <div>
+    <k-textbox
+      v-model="model1"
+      label="Sample text field"
+      :disabled="false"
+      :required="true"
+      :invalid="isInvalid"
+      invalid-text="The input is invalid"
+      :autofocus="true"
```

Comment:
Setting this to `true` causes the style guide page to auto-scroll down when it's loaded.